repo_health/index/static/client/src/components/issue-stats/issue-stats.component.js

```diff
@@ -0,0 +1,47 @@
+/*
+* issue-stats.component.js - (C) Copyright - 2017
+* This software is copyrighted to contributors listed in CONTRIBUTIONS.md.
+*
+* SPDX-License-Identifier: MIT
+*
+* Author(s) of this file:
+* J.Harding
+*
+* Create issue-stats component.
+*/
+
+import template from '../utils/base.template';
+
+const issueStatsComponent = {
+  template,
+  bindings: {
+    issueStatsUrl: '='
+  },
+  controller: class issueStatsComponent {
+    stats = null;
+    loadingStats = true;
+    loadingMsg = "Loading issue stats...";
```

Comment:
Little nit: Double quotes are only used if you are using single quotes inside of a string. The standard is single quotes.